drivers/console/Kconfig

```diff
@@ -163,6 +163,34 @@ config RTT_TX_RETRY_IN_INTERRUPT
 
 endif
 
+config SEMIHOSTING_CONSOLE
+	bool
+	depends on ARM
+	prompt "Use semihosting console"
+	select CONSOLE_HAS_DRIVER
+	help
+	  Enable this option to emit console messages over a debugger with
```

Comment:
This console driver is output-only, right? I wonder, if this fact should be made (more) explicit in the description.

Reply:
I think that this comes from the "semihosting" part - altough you can technically do input over semihosting (and other [weird stuff](https://github.com/ntfreak/openocd/blob/master/src/target/semihosting_common.c#L1024)), I don't think that's a commonly expected functionality.